src/stores/entities/annotations.go

```diff
@@ -1,9 +1,9 @@
 package entities
 
 type Annotation struct {
-	AWSKeyID             string `json:"aws_key_id,omitempty"`
-	AWSCustomKeyStoreID  string `json:"aws_custom_key_store_id,omitempty"`
-	AWSCloudHsmClusterID string `json:"aws_cloud_hsm_cluster_id,omitempty"`
-	AWSAccountID         string `json:"aws_account_id,omitempty"`
-	AWSArn               string `json:"aws_arn,omitempty"`
+	AWSKeyID             string `json:"AWSKeyID,omitempty"`
```

Comment:
If the json name is the same of the Go struct field, you don't have to specify it. You can just put `json:",omitempty"`

Reply:
This may be personal but IMO it's better to always name